history-of-the-quakers-v2/original/03-book-3-chapter-9.adoc

```diff
@@ -1,18 +1,19 @@
 == Chapter IX. NEW ENGLAND.
```

Comment:
Can you un-capitalize the `NEW ENGLAND` here? A lot of the chapters might be like that, so you might want to make a note to check them for each chapter as you get to them.  👍